app/src/main/java/de/westnordost/streetcomplete/quests/pitch_lit/AddPitchLit.kt

```diff
@@ -0,0 +1,38 @@
+package de.westnordost.streetcomplete.quests.pitch_lit
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.meta.updateWithCheckDate
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.ktx.toYesNo
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+
+class AddPitchLit : OsmFilterQuestType<Boolean>() {
+
+    override val elementFilter = """
+        ways with (leisure=pitch or leisure=track)
+        and (access !~ private|no)
+        and indoor != yes and !building
```

Comment:
Maybe `(!building or building = no)`?

`building=no` is [used around 10k times](https://taginfo.openstreetmap.org/tags/building=no).

Reply:
Thanks. [At least 300](https://overpass-turbo.eu/s/162f) of those are `leisure=pitch`. Should I update https://github.com/streetcomplete/StreetComplete/blob/a807e450698956ef6bf70eb2d1d376005656c03f/app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddPitchSurface.kt#L29 as well?

Reply:
Done.

Reply:
> building=no is used around 10k times.

I think that it can be safely ignored, given that it is not increasing ( https://taginfo.openstreetmap.org/tags/building=no#chronology ) and we have over 400 000 000 buildings increasing constantly - https://taginfo.openstreetmap.org/keys/building#chronology